rllib/offline/off_policy_estimator.py

```diff
@@ -53,21 +56,21 @@ def estimate(self, batch: SampleBatchType):
         raise NotImplementedError
 
     @DeveloperAPI
-    def action_prob(self, batch: SampleBatchType) -> TensorType:
+    def action_prob(self, batch: SampleBatchType) -> np.ndarray:
         """Returns the probs for the batch actions for the current policy."""
 
         num_state_inputs = 0
         for k in batch.keys():
             if k.startswith("state_in_"):
                 num_state_inputs += 1
         state_keys = ["state_in_{}".format(i) for i in range(num_state_inputs)]
-        log_likelihoods = self.policy.compute_log_likelihoods(
+        log_likelihoods: TensorType = self.policy.compute_log_likelihoods(
```

Comment:
We could actually make `compute_log_likelihoods()` return np.ndarray as well (actually, it is already doing this in TF graph mode --- just need to change it for eager_tf_policy.py and torch_policy.py).

Since we never need to differentiate through compute_log_likelihoods, it seems just a historical accident it returns TensorType.